### ISSUE TITLE
[RyuJIT/arm32] NYI_ARM: struct return

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/coreclr#11833]
For example:
```
HFATest.TestMan:Get_HFA02():struct - NYI (c:\gh\coreclr\src\jit\codegenarm.cpp:297 - NYI_ARM: struct return)
```
I see this in the following functions in the test tree:
```
<>c:<.cctor>b__3_0(struct):struct:this
<>c:<.cctor>b__3_1(struct):struct:this
<>c:<.cctor>b__3_3(struct):struct:this
<>c:<.cctor>b__3_4(struct):struct:this
<>c:<.cctor>b__3_6(struct):struct:this
<>c:<.cctor>b__3_7(struct):struct:this
Color:Plus(struct,struct):struct
Color:Times(struct,struct):struct
HFATest.TestMan:Get_HFA02():struct
HFATest.TestMan:Get_HFA03():struct
HFATest.TestMan:Identity_HFA02(struct):struct
HFATest.TestMan:Identity_HFA03(struct):struct
Plane:Normal(struct):struct:this
RayTracer:GetPoint(double,double,ref):struct:this
Sphere:Normal(struct):struct:this
Test.Program:getTestValue(float,float,float):struct
Test01.Program:RandomSimpleVector3():struct
Test01.Program:RandomVector3():struct
Test01.Program:RandomWrappedVector3():struct
```
(Unfortunately, my test methodology doesn't give me an easy mapping back to test name.)

**Comment by BruceForstall** [comment]
fyi @hseok-oh

**Comment by hseok-oh** [comment]
@BruceForstall Could we close this issue? I think it was fixed by #12451
